pkg/core/object/replicate.go

```diff
@@ -38,14 +38,46 @@ const (
 //	"validuntil": last valid block number for meta information
 //
 // Last valid epoch is object's creation epoch + 10.
-func EncodeReplicationMetaInfo(cID cid.ID, oID, firstPart oid.ID, pSize uint64,
-	deleted, locked []oid.ID, vub uint64, magicNumber uint32) []byte {
+func EncodeReplicationMetaInfo(o object.Object, vub uint64, magicNumber uint32) ([]byte, error) {
```

Comment:
callers aint happy about this change. One was completely okay w/o an error, while the second forced to do multiple decoding. If u wanna decrease prm num, typedef a structure

Reply:
> typedef a structure

object.Object is such a struct, why not?

Reply:
object has no `deleted, locked []oid.ID` fields

Reply:
> object has no deleted, locked []oid.ID fields

yes, but `object.Object` is enough to get this information and it will be always gotten like this

this commit was already dropped anyway